lib/actions/apiV2.js

```diff
@@ -962,7 +922,7 @@ export function routingQuery(searchId = null, updateSearchInReducer) {
               const withCollapsedShortNames =
                 response.data?.plan?.itineraries?.map((itin) => ({
                   ...itin,
-                  legs: itin.legs?.map(processLeg)
+                  legs: itin.legs?.map(convertGraphQLResponseToLegacy)
```

Comment:
This is great. Could you also update the dependencies to use core-utils 11.0.2 when it gets published?

Reply:
Sure, makes sense to roll that into this